proxy/m2_actions.c

```diff
@@ -26,6 +26,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "common.h"
 #include "actions.h"
 
+
+static char*
```

Comment:
The return type should be `gchar*` since the return string is allocated with a GLib-2.0 function, at least to give the hint that it must also be freed with a GLib-2.0 function.

Reply:
I saw it, then forgot. It's fixed now.